app/views/customers/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Name: <%= @customer.name %></h1>
+
+<h3>Items: </h3>
+<% @customer.items.each do |item| %>
+  <p>Item name: <%= item.name %></p>
+  <p>Item price: <%= item.price %></p>
+  <p>Supermarket: <%= item.supermarket_id %></p>
+<% end %>
+
+<h3>Add an item to this list</h3>
+  <%= form_with url: "/customers/#{@customer.id}", method: :get, data: { turbo: false } do |form| %>
```

Comment:
We probably want this to be a POST - we are *creating* a joins record between this customer and a given item. The most RESTful option would be POSTing to `/customers/:id/items`

Reply:
Be careful -- it's not really a search but rather the user already knows the ID and wants to associate it with this customer